src/ZAxis.re

```diff
@@ -10,7 +10,7 @@ external make:
     ~allowDuplicatedCategory: bool=?,
     ~axisLine: 'axisLine=?,
     ~className: string=?,
-    ~dataKey: string=?,
+    ~dataKey: 'dataKey=?,
```

Comment:
What other types can be `dataKey`? Is it really anything? Is it worth documenting + including a link to official docs if they exist?

Reply:
This is why I put this back into draft actually.

I've been hitting a lot of bindings here that are marked as anything but they really aren't so I've been replacing them with `[@mel.unwrap]` versions in my local copy and I'll update this PR when I have that in a good spot and confirmed they are all indeed working as intended.